crates/rolldown/tests/fixtures/warnings/ambiguous_external_namespace/artifacts.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+assertion_line: 176
+expression: content
+input_file: crates/rolldown/tests/fixtures/warnings/ambiguous_external_namespace
+---
+# warnings
+
+## AMBIGUOUS_EXTERNAL_NAMESPACES
```

Comment:
https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00entry.js%00import+%7B+foo+%7D+from+%27.%2Fproxy%27%0Aconsole.log%28foo%29&b=%00a.js%00export+const+foo+%3D+1&b=%00b.js%00export+const+foo+%3D+2&b=%00proxy.js%00export+*+from+%27.%2Fa%27%0Aexport+*+from+%27.%2Fb%27&o=--bundle+--splitting+--format%3Desm+--external%3Aext+--outdir%3Ddist

Esbuild treats this as an error and has better report. Let's merge this PR for now. I'll open an new issue to port esbuild's behavior.